packages/suite/src/views/wallet/tokens/index.tsx

```diff
@@ -17,15 +19,17 @@ export const Tokens = () => {
 
     const selectedAccount = useSelector(state => state.wallet.selectedAccount);
     const dispatch = useDispatch();
+    const routeName = useSelector(selectRouteName);
 
     useEffect(() => {
         if (
             selectedAccount.status === 'loaded' &&
-            !selectedAccount.network?.features.includes('tokens')
+            !hasNetworkFeatures(selectedAccount.account, 'tokens') &&
+            routeName !== 'wallet-index'
```

Comment:
this fixes the issue, but the problem now is that there flashes "no data" for a page of a coin that doesn't have tokens page. But I guess that's fixable